packages/web/src/index.html

```diff
@@ -374,6 +376,19 @@ <h4 class="mt-500 mb-300">Variable width icons (auto width)</h4>
       <gcds-icon name="fire" size="text"></gcds-icon>
       <gcds-icon name="utensils" size="text"></gcds-icon>
 
+      <!-- ------------- Link ------------- -->
+      <h3 class="mt-700 mb-400 bb-sm" id="links">Link</h3>
+      <div class="my-200"><gcds-link>Base link</gcds-link></div>
+      <div class="my-200">This is a <gcds-link size="small" href="/">small link</gcds-link> if you compare it to regular text.</div>
+      <div class="my-200"><gcds-link href="https://picsum.photos/480/270" external>This is an external link</gcds-link></div>
+      <div class="my-200">This is a <gcds-link href="https://picsum.photos/200/200.jpg" download="random_image" type="image/jpeg">download link (PDF, 1.3 MB)</gcds-link> with type indicated</div>
+      <div class="my-200">Contact us <gcds-link href="mailto:test@test.com?subject=Test%20email">via email</gcds-link> or by phone.</div>
+      <div class="my-200">Contact us <gcds-link href="tel:1234567890">by phone</gcds-link> or by email.</div>
+      <div class="my-200">There is text before and <gcds-link href="/" display="inline-block">(sample link)</gcds-link> after this link.</div>
+      <div class="my-200"><gcds-link href="/" referrerpolicy="no-referrer" external>This is a sample link with a referrerpolicy</gcds-link></div>
+      <div class="my-200"><gcds-link href="" onClick='window.alert("Thank you!")'>Click me!</gcds-link></div>
```

Comment:
This is suuuuuper nit-picky so apologies about that but can you update all these div's to use p tags instead since these are all text-based links? Trying to use semantic tags whenever possible :)

Reply:
Done! No worries about the nit